packages/unittest_runner/unittest_runner/validation_runner.py

```diff
@@ -0,0 +1,13 @@
+
+from unittest import TextTestResult
+
+class ValidationTestResult(TextTestResult):
+  # The default behavior is to display the test name and short description.
+  # We want to display only the short description if it exists, otherwise just the test name.
+  # Students don't see the tests, so we don't need to display the test name.
```

Comment:
Matter of taste for sure, but I personally wouldn't be shy about linking back to this PR to show the before/after in case somebody is trying to understand in the future.

Reply:
good idea!